posts/2023/july/linguist/2023-07-13-linguist.md

```diff
@@ -13,58 +13,58 @@ image:
 
 ![pageTranslation.png](./pageTranslation.png)
 
 ## Use your favorite translation service
 
-Linguist have few embedded translators, include google translator, and you can add your favorite translator to linguist yourself with [custom translators](https://github.com/translate-tools/linguist/blob/bcc8c471fdede42b6bbc541144eb89a93587f605/docs/CustomTranslator.md) button. I really like this powerful feature, with Linguist you are not locked on Google Translator anymore!
+Linguist has a few embedded translators, include google translator, and you can add your favorite translator to linguist yourself with [custom translators](https://github.com/translate-tools/linguist/blob/bcc8c471fdede42b6bbc541144eb89a93587f605/docs/CustomTranslator.md) button. I really like this powerful feature - with Linguist you are not locked to using Google Translator anymore!
 
 Look, you can just insert any JavaScript code that implement translator interface:
 
 ![custom_translator.png](./custom_translator.png)
 
 ## Keep your privacy
 
-Start of version 5.0, Linguist includes embedded offline translator, based on [Bergamot project](https://github.com/browsermt/bergamot-translator) - mozilla's machine translation project, compiled to WebAssembly.
+Starting from version 5.0, Linguist includes embedded offline translator, based on [Bergamot project](https://github.com/browsermt/bergamot-translator) - mozilla's machine translation project, compiled to WebAssembly.
 
 ![settings.png](./settings.png)
 
-With offline translator, you will not sent your private texts over internet at all. Translation runs on your device locally, you still needs internet to download AI models for your languages direction once, then you can even off your internet connection and continue to translate texts.
+With the offline translator, you will not be sending your private texts over internet at all! The translation runs on your device locally once you have downloaded the AI models for the languages you want translate to and from.
 
 Unlike other browser extensions, Linguist is not just wrapper over Google Translator Widget, so all translation features are available offline with offline translator, thus you can use all Linguist features and stay private.
 
 ## Learn the languages
 
-With dictionary you can bookmark translations and find it later, when you need those expressive words you've found. I like this feature well, my dictionary is about 800+ translations. Of course, there you can search translations with fuzzy search.
+With the dictionary feature, you can bookmark translations and find them later, when you need those expressive words/texts you found earlier. I like this feature as well and my dictionary has about 800+ translated texts already. Of course, you can search in your dictionary with a fuzzy search.
```

Comment:
> words/texts
Let's choose one word, to be consistent

Reply:
I accidentally opened a new pull request. Please check the corrections and merge. Sorry for the trouble. Still dont have a hang of how github works

Reply:
I had fix on this PR. Thanks for help, i will close another PR